cli/src/cli.rs

```diff
@@ -1018,11 +1018,21 @@ fn process_airdrop(
         faucet_addr
     );
 
-    request_and_confirm_airdrop(&rpc_client, faucet_addr, &pubkey, lamports, &config)?;
+    let pre_balance = rpc_client.get_balance(&pubkey)?;
+
+    let signature =
+        request_and_confirm_airdrop(&rpc_client, faucet_addr, &pubkey, lamports, &config)?;
+    println!("{}", signature);
 
     let current_balance = rpc_client.get_balance(&pubkey)?;
 
-    Ok(build_balance_message(current_balance, false, true))
+    if current_balance < pre_balance.saturating_add(lamports) {
+        println!("Balance unchanged");
+        println!("Run `solana confirm -v <SIGNATURE>` for more info");
```

Comment:
wdyt about reducing the copy-paste for the user?
```suggestion
        println!("Run `solana confirm -v {}` for more info", signature);
```

Reply:
Heh, I attempted that at first, but cli's `request_and_confirm_airdrop()` returns a formatted string (`config.output_format.formatted_string(&CliSignature { .. })`), and I got sidetracked/gave up when it didn't Just Work.

Thanks for the encouragement; I'll do the extra surgery 🙂

Reply:
✔️ New cli output:
```
$ solana airdrop 2 --faucet-port 9901`
Requesting airdrop of 2 SOL from 127.0.0.1:9901

Signature: 8ohk7nytJ4UJQfvTN6uXpVj9PxxqokVVshWudapDPYyH1YgPbEFJkogUrnvJgVojSibBK1cy19oVfwPif1Ws1Zd

Balance unchanged
Run `solana confirm -v 8ohk7nytJ4UJQfvTN6uXpVj9PxxqokVVshWudapDPYyH1YgPbEFJkogUrnvJgVojSibBK1cy19oVfwPif1Ws1Zd` for more info
```

Reply:
👍🏼